src/index.js

```diff
@@ -458,14 +472,26 @@ Consider using the serverless-python-requirements plugin to help you package Pyt
 		const handlerModulePath = func.handler.substr(0, func.handler.lastIndexOf("."));
 		// e.g. functions/hello.world.handler -> handler
 		const handlerFuncName = handler.substr(handler.lastIndexOf(".") + 1);
-		const wrappedFunction = `
+
+        
+
+		const wrappedESMFunction = `
+import lumigo from '@lumigo/tracer'
+import {${handlerFuncName} as lumigoHandler} from '../${handlerModulePath}.${this.nodeModuleFileExtension}'
+const tracer = lumigo(${this.getNodeTracerParameters(token, options)})
```

Comment:
This code doesn't work. 
Did you meant?
```suggestion
const tracer = lumigo({ ${this.getNodeTracerParameters(token, options)}})
```
Can you please verify that your code works in your scenario?